src/main/java/org/testng/TestNG.java

```diff
@@ -1577,8 +1555,6 @@ public void configure(Map cmdLineArgs) {
     result.xmlPathInJar = (String) cmdLineArgs.get(CommandLineArgs.XML_PATH_IN_JAR);
     result.junit = (Boolean) cmdLineArgs.get(CommandLineArgs.JUNIT);
     result.mixed = (Boolean) cmdLineArgs.get(CommandLineArgs.MIXED);
-    result.master = (String) cmdLineArgs.get(CommandLineArgs.MASTER);
```

Comment:
Some warnings/exceptions are missing here to say that the master/slave feature moved into another jar.